newdle/client/src/components/ParticipantTable.js

```diff
@@ -2,10 +2,9 @@ import React, {useState} from 'react';
 import moment from 'moment';
 import {useSelector} from 'react-redux';
 import {Radio, Icon, Label, Table} from 'semantic-ui-react';
-import {CircularProgressbar, buildStyles} from 'react-circular-progressbar';
```

Comment:
Could you `npm remove` it (while inside `newdle/client`) so we don't have it as a dependency in `package{,-lock}.json` anymore?